Projects/Metatron_Protocol_Launch/.~lock.Metatron_Plan_v.0.1.ods#

```diff
@@ -0,0 +1 @@
+,flaxscrip,flaxlap,01.10.2021 13:25,file:///home/flaxscrip/.config/libreoffice/4;
```

Comment:
Probably don't want to commit this file?

Reply:
Agreed.